es-core/src/components/IList.h

```diff
@@ -191,10 +191,6 @@ class IList : public GuiComponent
 	{
 		PowerSaver::setState(velocity == 0);
 
-		// generate an onCursorChanged event in the stopped state when the user lets go of the key
-		if(velocity == 0 && mScrollVelocity != 0)
-			onCursorChanged(CURSOR_STOPPED);
```

Comment:
I want to make sure that removing this didn't break anything else.

The call to `stopScrolling()` would also call `onCursorChanged()` so that is where it would be called once, but believe there are other places that only call `listInput(0);`